docs/developers/new_model_to_armory.md

```diff
@@ -0,0 +1,109 @@
+# How to add a new model into armory
+In this file, I will show 3 different examples of how to add a model into armory-library. I will show how to use a model from HuggingFace, how to use a model from github, and how to use a model from a different python library.
+
+Common imports shared acrossed all examples
+```python
+from charmory.model.image_classification import JaticImageClassificationModel
+from charmory.track import track_init_params, track_params
+
+from art.estimators.classification import PyTorchClassifier
+
+import torch
+from transformers import AutoImageProcessor, AutoModelForImageClassification
+```
+In each example the model and classifier objects are created to be used by armory-library
+
+## Example 1: Using model from HuggingFace
+This example using built in armory library capabilities since we support direct ability to use model from HuggingFace
+```python 
+model = JaticImageClassificationModel(
+    track_params(AutoModelForImageClassification.from_pretrained)(
+        "tianzhihui-isc/vit-base-patch16-224-in21k-finetuned-pokemon-classification"
+    ),
+)
+
+classifier = track_init_params(PyTorchClassifier)(
+    model,
+    loss=torch.nn.CrossEntropyLoss(),
+    optimizer=torch.optim.Adam(model.parameters(), lr=0.003),
+    input_shape=(3, 224, 224),
+    channels_first=True,
+    nb_classes=150,
+    clip_values=(-1, 1),
+)
+```
+For this section of code we create two variables model and classifier. We found a model on HuggingFace that is trained on the same pokemon image dataset with the model card 'tianzhihui-isc/vit-base-patch16-224-in21k-finetuned-pokemon-classification'. This can be replaced with another model off of Huggingface.
```

Comment:
Thanks for help! I got this from the other PR for the colab file since I am running the same lines as that example. I can make these changes in that PR as well b/c the same issues should be there.